src/views/CodeOfConduct.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="row">
+    <div class="col-xs-12 col-md-10 col-md-offset-1">
+      <h1>Code of Conduct</h1>
+      <p>
+        Baltimore Code and Coffee is dedicated to providing a harassment-free experience for everyone, regardless of
+        level
+        of experience, gender, gender identity and expression, sexual orientation, disability, physical appearance, body
+        size, race, age, or religion. We do not tolerate harassment of group participants in any form. Sexual language
+        and
+        imagery is not appropriate for any event or communication channel. Group participants violating these rules may
+        be
+        sanctioned or expelled from the event at the discretion of the organizers.
+      </p>
+      <p> Harassment includes, but is not limited to:</p>
+      <ul>
+        <li>Verbal comments that reinforce social structures of domination related to gender, gender identity and
+          expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or level of
+          experience
+        </li>
+        <li>Sexual images in public spaces</li>
+        <li>Deliberate intimidation, stalking, or following</li>
+        <li>Harassing photography or recording</li>
+        <li>Sustained disruption of discussion or other events</li>
+        <li>Inappropriate physical contact</li>
+        <li>Unwelcome sexual attention</li>
+        <li>Advocating for, or encouraging, any of the above behaviour</li>
+      </ul>
+
+      <h2>Enforcement</h2>
+
+      <p>
+        Participants asked to stop any harassing behavior are expected to comply immediately. If a participant engages
+        in
+        harassing behaviour, event organisers retain the right to take any actions to keep the event a welcoming
+        environment
+        for all participants. This includes warning the offender or expulsion from the event.
+      </p>
+
+      <p> We expect participants to follow these rules at all event venues and event-related social activities. </p>
+
+      <h2>Reporting</h2>
+
+      <p>
+        If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. Group
+        organizers
+        can be identified on Meetup. Harassment and other code of conduct violations reduce the value of our event for
+        everyone. We want you to be happy at our event. People like you make our event a better place.
+      </p>
+
+      <p> You can make a report either personally or anonymously. </p>
+
+      <h2>Anonymous Report</h2>
+
+      <p>
+        You can make an
+        <RouterLink :to="{name: 'report'}">anonymous report</RouterLink>
```

Comment:
In the future it might be good to include this link in other places as well so it's not only in the code of conduct.